javascripts/main.js

```diff
@@ -16,7 +16,7 @@ define([
   var getFilterUrl = function() {
     return location.href.indexOf('/#/filters') > -1
       ? location.href
-      : location.href + 'filters';
+      : `${location.href}${filters}`;
```

Comment:
I think there's a subtle bug here because we dropped the quote marks around `filters`, turning it into a local variable and causing this error when you click on any tags:

<img width="377"  src="https://user-images.githubusercontent.com/359239/66488806-f47e2080-ea84-11e9-95c4-1c91e6854791.png">

I think this is what it should be, but I'm not 100% sure:

```suggestion
      : `${location.href}filters`;
```

Reply:
Not sure how I missed this... thanks